dockstore-client/src/main/java/io/dockstore/client/cli/nested/AbstractEntryClient.java

```diff
@@ -636,28 +652,33 @@ private void launchWdl(final List<String> args) {
         final SourceFile wdlFromServer;
         try {
             // Grab WDL from server and store to file
-            wdlFromServer = getDescriptorFromServer(entry, "wdl");
             final File tempDir = Files.createTempDir();
-            final File tempWdl = File.createTempFile("temp", ".wdl", tempDir);
-            Files.write(wdlFromServer.getContent(), tempWdl, StandardCharsets.UTF_8);
-            downloadDescriptors(entry, "wdl", tempDir);
-
-            Pattern p = Pattern.compile("^import\\s+\"(\\S+)\"(.*)");
-            File file = new File(tempWdl.getAbsolutePath());
-            List<String> lines = FileUtils.readLines(file);
-            File tmp = new File(tempDir + File.separator + "overwrittenImports.wdl");
-
-            // Replace relative imports with absolute (to temp dir)
-            for (String line : lines) {
-                Matcher m = p.matcher(line);
-                if (!m.find()) {
-                    FileUtils.writeStringToFile(tmp, line + "\n", true);
-                } else {
-                    if (!m.group(1).startsWith(File.separator)) {
-                        String newImportLine = "import \"" + tempDir + File.separator + m.group(1) + "\"" + m.group(2) + "\n";
-                        FileUtils.writeStringToFile(tmp, newImportLine, true);
+            File tmp;
+            if (!isLocalEntry) {
+                wdlFromServer = getDescriptorFromServer(entry, "wdl");
+                File tempWdl = File.createTempFile("temp", ".wdl", tempDir);
+                Files.write(wdlFromServer.getContent(), tempWdl, StandardCharsets.UTF_8);
+                downloadDescriptors(entry, "wdl", tempDir);
+
+                Pattern p = Pattern.compile("^import\\s+\"(\\S+)\"(.*)");
+                File file = new File(tempWdl.getAbsolutePath());
+                List<String> lines = FileUtils.readLines(file);
+                tmp = new File(tempDir + File.separator + "overwrittenImports.wdl");
+
+                // Replace relative imports with absolute (to temp dir)
```

Comment:
Could you add into a comment why this is done?
i.e. why run with absolute imports rather than create a temp directory to run in with the imports in (relatively) the correct locations